sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -619,17 +610,16 @@ def delete_container(
                 :dedent: 12
                 :caption: Deleting a container in the blob service.
         """
-        container = self.get_container_client(container) # type: ignore
```

Comment:
Again, not sure if we can just remove it (or did your type checker mark these as unnecessary?)

Reply:
yup, my type checker told me those are unnecessary comments